ml-prims/src/functions/logisticReg.h

```diff
@@ -89,12 +81,6 @@ void logisticRegLossGrads(math_t *input, int n_rows, int n_cols,
 	    if (pen_grads != NULL)
 	        CUDA_CHECK(cudaFree(pen_grads));
```

Comment:
Why is this `cudaFree` still there? Oversight?

Reply:
Good catch! This was oversighted. Thanks!